packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -756,27 +703,23 @@ describe( 'Default search suggestions', () => {
 	it( 'should display a list of initial search suggestions when there is no search value or suggestions', async () => {
 		const expectedResultsLength = 3; // Set within `LinkControl`.
 
-		act( () => {
-			render( <LinkControl showInitialSuggestions />, container );
-		} );
+		const { container } = render( <LinkControl showInitialSuggestions /> );
 
 		await eventLoopTick();
 
 		// Search Input UI.
 		const searchInput = getURLInput();
 
-		const searchResultsWrapper =
-			container.querySelector( '[role="listbox"]' );
-		const initialSearchResultElements =
-			searchResultsWrapper.querySelectorAll( '[role="option"]' );
+		const searchResultsWrapper = screen.queryByRole( 'listbox' );
```

Comment:
The `labelledby` check can be covered in a less implementation-specific way by a query like:

```suggestion
		const searchResultsWrapper = screen.queryByRole( 'listbox', {
			name: 'Recently updated',
		} );
```

Reply:
Done in c689763e83840e41215fda710fa467b34f4dc539 where I addressed a few more instances of this.